tests/console/commands/test_new.py

```diff
@@ -55,10 +57,11 @@ def verify_project_directory(
     else:
         package_include = {"include": package_path.parts[0]}
 
+    name = poetry.local_config.get("name")
     packages = poetry.local_config.get("packages")
 
     if not packages:
-        assert poetry.local_config.get("name") == package_include.get("include")
+        assert module_name(name) == package_include.get("include") if name else ""
```

Comment:
```suggestion
        assert module_name(name) == package_include.get("include")
```

`name` is always set isn't it?

If I don't miss anything the expression wouldn't work if it wasn't set because there is a missing bracket. It had to be:

```
assert module_name(name) == (package_include.get("include") if name else "")
```

Reply:
`name` is always set because `module_name()` expects a `str`. Therefore your suggestion does not pass mypy.

However, I think this may be a better approach. What do you think?
```python
    name = poetry.local_config.get("name", "")
    packages = poetry.local_config.get("packages")

    if not packages:
        assert module_name(name) == package_include.get("include")
```

Reply:
LGTM 👍